packages/cli/lib/lib/webpack/create-load-manifest.js

```diff
@@ -8,7 +8,7 @@ module.exports = (assets, isESMBuild = false, namedChunkGroups) => {
 		styles = [];
 	for (let filename in assets) {
 		if (!/\.map$/.test(filename)) {
-			if (/route-/.test(filename)) {
+			if (/^route-.*\.js$/.test(filename)) {
```

Comment:
This created some junk output in `push-manifest.json` when ESM is disabled, like in our test suite. Wouldn't have ever been a problem, as these junk "routes" would never match to a URL when prerendering, but certainly looked odd.

```json
{
  "/": {
    "bundle.2da73.css": {
      "type": "style",
      "weight": 1
    },
    "bundle.44866.js": {
      "type": "script",
      "weight": 1
    },
    "route-home.chunk.3cec8.js": {
      "type": "script",
      "weight": 0.9
    },
    "route-home.chunk.bcb8a.css": {
      "type": "style",
      "weight": 0.9
    }
  },
  "/.chunk.bcb8a.css": {
    "bundle.2da73.css": {
      "type": "style",
      "weight": 1
    },
    "bundle.44866.js": {
      "type": "script",
      "weight": 1
    },
    "route-home.chunk.bcb8a.css": {
      "type": "script",
      "weight": 0.9
    }
  },
  "/profile.chunk.6dd80.css": {
    "bundle.2da73.css": {
      "type": "style",
      "weight": 1
    },
    "bundle.44866.js": {
      "type": "script",
      "weight": 1
    },
    "route-profile.chunk.6dd80.css": {
      "type": "script",
      "weight": 0.9
    }
  },
  "/profile": {
    "bundle.2da73.css": {
      "type": "style",
      "weight": 1
    },
    "bundle.44866.js": {
      "type": "script",
      "weight": 1
    },
    "route-profile.chunk.ddf94.js": {
      "type": "script",
      "weight": 0.9
    },
    "route-profile.chunk.6dd80.css": {
      "type": "style",
      "weight": 0.9
    }
  }
}
```